backend/app/routers/metadata_files.py

```diff
@@ -248,6 +248,20 @@ async def update_file_metadata(
     Returns:
         Metadata document that was updated
     """
+
+    # check if metadata with file version exists, replace metadata if none exists
+    if (
+        version_md := await db["metadata"].find_one(
+            {
+                "_id": ObjectId(metadata_in.metadata_id),
+                "resource.resource_id": ObjectId(file_id),
+                "resource.version": metadata_in.file_version,
```

Comment:
It's necessary because I need to find out if metadata with the specific file id and version id exists. If it does, we can simple update the metadata here in this method, else I need to call 'replace_file_metadata' where it creates a new object with the file id  and version. I could have updated the current method but chose to re-use what we already have.